extensions/jms/src/main/java/org/mule/extensions/jms/internal/connection/provider/activemq/ActiveMQConnectionProvider.java

```diff
@@ -63,9 +63,9 @@ private void createConnectionFactory() throws ActiveMQConnectionException {
       if (JMS_2_0.equals(getConnectionParameters().getSpecification())) {
 
         //TODO we could support a JMS 2.0 default using ActiveMQ Artemis (HornetQ) instead of ActiveMQ 5.x
-        throw new IllegalArgumentException(
-                                           "No ConnectionFactory was provided, but JMS 2.0 specification was selected."
-                                               + " Default ActiveMQConnectionFactory implementation provides support only for JMS 1.1 and 1.0.2b versions");
+        throw new ActiveMQException(
```

Comment:
mmm... mixed feelings here, the IllegalArgument seems to be more clear over what the error was. @estebanwasinger do we have some kind of generic "INVALID_CONFIGURATION" errorType? 
@marianogonzalez opinions/ideas?